lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -614,6 +627,7 @@ interface SdkEvent {
     PaymentRefundPending(Payment details);
     PaymentSucceeded(Payment details);
     PaymentWaitingConfirmation(Payment details);
+    PaymentWaitingUserReview(Payment details);
```

Comment:
When such event is emitted and the payment state is changed to WaitingUserAction it seems that the next step is for the user to get a quote from the swapper (e.g the new fees) right?
In that case, since fees are changing all the time, we probably need an API to get the current fees.
Should we add such API?

Reply:
Not related to the interface here but we probably also need to subscribe to the swapper webhook in such cases ( I think it is `TransactionLockupFailed` event) so the mobile user will get a notification in such cases.

Reply:
I've added a new method for fetching the current fees. I had misunderstood the process before.

I'll work on the notification plugins to show a notification when user action is required.